.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+default_stages: [commit, push]
+minimum_pre_commit_version: 2.10.1
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-ast
+    -   id: check-docstring-first
+    -   id: check-merge-conflict
+    -   id: end-of-file-fixer
+    -   id: no-commit-to-branch
+        args: [--branch, master]
+    -   id: trailing-whitespace
+
+-   repo: local
+    hooks:
+    -   id: mypy
+        name: Run mypy
+        entry: python -m mypy
+        language: system
+        types: [python]
+        stages: [push, commit]
+
+    -   id: autoflake
```

Comment:
this is useful package, I use it myself. But using it as a pre-commit hook can lead to issues, since sometimes you have to import stuff and not use it directly (you might only need side effects of imports). so just let `flake` to complain about imports and clean them locally